main.go

```diff
@@ -296,18 +296,10 @@ func main() {
 	}
 
 	registerControllers(ctx, mgr)
-	// +kubebuilder:scaffold:builder
 
-	if err := mgr.AddReadyzCheck("webhook", mgr.GetWebhookServer().StartedChecker()); err != nil {
```

Comment:
@reviewers, please pay close attention here. I believe we were duplicating some code so I removed it, but let me know if there is a good reason for this being called twice (here and line 520)

same thing with mgr.Start (line 304 and line 514)